matico_components/src/Stores/MaticoSpecSlice.tsx

```diff
@@ -45,7 +90,7 @@ export const stateSlice = createSlice({
       }>
     ) => {
       state.currentEditPath = action.payload.editPath;
-      state.currentEditType = action.payload.editType;
+      state.currentEditType = extractEditType(action.payload.editPath);
```

Comment:
@stuartlynn Following the thread on inner/outer, probably swap this back to be explicit?

Reply:
Humm perhaps this is worth thinking about some more. Outer tagging would look like 


```js
{
  Map: {
    lat : 2,
    lng:2 
  }
}  
```
so the extraction would look like 

```
Object.keys( _get(state,editpath))[0]
```
If we use the convention that the edit path just points to the array entry.

If we use inner tagging it would be something like 

```js
{
type:"map",
lat: 2,
lng:3
}
```

Then extracting the type is a bit easier. 

Lets leave it as it is just now and fix when we decide on what convention to use.